common/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tantivy-common"
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
I think it's better to let the versioning handle by cargo release, to keep everything in sync